modules/end-user-guide/pages/microsoft-visual-studio-code-open-source-ide.adoc

```diff
@@ -0,0 +1,30 @@
+:_content-type: CONCEPT
+:description: Microsoft Visual Studio Code - Open Source IDE
+:keywords: user-guide, selecting, IDE, select, selecting, editor, ide, vs-code, visual-studio, visual-studio-code, jetbrains, intellij, pycharm, theia, che-theia
+:navtitle: Microsoft Visual Studio Code - Open Source
+:page-aliases:
+
+[id="microsoft-visual-studio-code-open-source-ide"]
+= Microsoft Visual Studio Code - Open Source IDE
+
+The {prod-short} build of link:https://github.com/microsoft/vscode[Microsoft Visual Studio Code - Open Source] is the default cloud IDE of a new workspace.
+
+You can have extensions automatically installed in this IDE for workspaces, as explained in the next section, from the xref:administration-guide:extensions-for-microsoft-visual-studio-code-open-source.adoc[Open VSX registry].
+
+TIP: You can use the link:https://code.visualstudio.com/api/ux-guidelines/command-palette[Command Palette] to find and run the commands specified in `devfile.yaml`.
+
+[TIP]
+====
+
+You can use *{prod2}* commands by clicking *{prod2}* in the link:https://code.visualstudio.com/api/ux-guidelines/status-bar[Status Bar] or finding them through the link:https://code.visualstudio.com/api/ux-guidelines/command-palette[Command Palette]:
+
+* *{prod2}: Open Dashboard*
+* *{prod2}: Stop Workspace*
+* *{prod2}: Restart Workspace*
+* *{prod2}: Open Documentation*
+
+====
+
+NOTE: You might see your organization's branding in this IDE if your organization customized it through a branded build.
```

Comment:
We have an issue to document VS Code for Che specific features https://github.com/eclipse/che/issues/21835. It should not be done in this PR. It deserve its own PR. But just FYI.

Reply:
Thank you for the heads up. I opened [RHDEVDOCS-5026](https://issues.redhat.com/browse/RHDEVDOCS-5026) to assign a technical writer when the developers are ready to start preparing more content on this topic.